packages/dnb-eufemia/src/components/accordion/AccordionContent.tsx

```diff
@@ -15,14 +14,19 @@ import {
 } from '../../shared/component-helper'
 import { useMediaQuery } from '../../shared'
 import AccordionContext from './AccordionContext'
-import {
-  spacingPropTypes,
-  createSpacingClasses,
-} from '../space/SpacingHelper'
+import { createSpacingClasses } from '../space/SpacingHelper'
 import HeightAnimation from '../height-animation/HeightAnimation'
+import { SpacingProps } from '../space/types'
+
+export type AccordionContentProps = React.HTMLProps<HTMLElement> &
+  SpacingProps & {
+    instance?: React.LegacyRef<any>
+    className?: string
+    children?: React.ReactNode | ((...args: any[]) => any)
+  }
 
-export default function AccordionContent(props) {
-  const context = React.useContext(AccordionContext)
+export default function AccordionContent(props: any) {
```

Comment:
Could this be typed using `AccordionContentProps`?

```suggestion
export default function AccordionContent(props: `AccordionContentProps`) {
```

Reply:
Yeah, nice catch 😄 🙏